desci-server/test/integration/data.test.ts

```diff
@@ -302,10 +300,10 @@ describe('Data Controllers', () => {
       //   expect(oldDataBucketCid).to.not.equal(newDataBucketCid);
       // });
       it('should have added a code component to the manifest', async () => {
-        console.log(res.body.manifest);
-        const handle = backendRepo.find(getAutomergeUrl(documentId));
-        const doc = await handle.doc();
-        console.log('Doc', doc);
+        console.log(res.body.manifest?.components);
+        // const handle = backendRepo.find(getAutomergeUrl(documentId));
+        // const doc = await handle.doc();
+        // console.log('Doc', doc.manifest.components);
```

Comment:
Is this test in debugging? Seeing the log below too

Reply:
yep